tests/Gedmo/Tree/MaterializedPathORMRepositoryTest.php

```diff
@@ -346,7 +347,12 @@ public function testIssue458()
         $this->em->persist($newNode);
         $this->em->flush();
 
-        $this->assertRegexp('/Food\-\d+,New\sNode\-\d+/', $newNode->getPath());
+        // @todo: Remove the condition and the `else` block when dropping support for "phpunit/phpunit" < 9.0.
+        if (version_compare(Version::id(), '9.0.0') >= 0) {
```

Comment:
I'd do `method_exists($this, 'assertMatchesRegularExpression')` here, it's generally more reliable than version checks (plus, the new method was actually introduced in PHPUnit 9.1, so if 9.0 were allowed to install then this would actually fail).

Reply:
Done.